robots/cmd/labels-checker/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/google/go-github/github"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+	"os"
+	"strings"
+)
+
+type options struct {
+	tokenPath           string
+	endpoint            string
+	org                 string
+	repo                string
+	author              string
+	branchName          string
+	ensureLabelsMissing string
+}
+
+func (o *options) Validate() error {
+	if o.org == "" {
+		return fmt.Errorf("org is required")
+	}
+	if o.repo == "" {
+		return fmt.Errorf("repo is required")
+	}
+	if o.author == "" {
+		return fmt.Errorf("author is required")
+	}
+	if o.branchName == "" {
+		return fmt.Errorf("branch-name is required")
+	}
+	return nil
+}
+
+func (o *options) GetEnsureLabelsMissing() []string {
+	return strings.Split(o.ensureLabelsMissing, ",")
+}
+
+func gatherOptions() options {
+	o := options{}
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
Why do you need a FlagSet here? I have not seen any sub-commands around.

It is also common to process the flags on `init`, supporting other potential flags from other packages. Only the `Parse` is called on main usually.
For that, a global `options` is needed, but it seems ok as there is only one possible options data collection.